modules/tinymce/CHANGELOG.md

```diff
@@ -19,10 +19,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 - Transparent elements, like anchors, are now allowed in the root of the editor body if they contain blocks. #TINY-9172
+- Colorswatch keyboard navigation now starts on currently selected color if present in the colorswatch. Otherwise defaults to current behavior #TINY-9283
```

Comment:
Since the default here is to select the first item I don't think we need to mention the current behavior.

```suggestion
- Colorswatch keyboard navigation now starts on currently selected color if present in the colorswatch.  #TINY-9283
```